packages/ui/IconButton/IconButton.js

```diff
@@ -30,6 +30,7 @@ import componentCss from './IconButton.module.less';
  * @memberof ui/IconButton
  * @ui
  * @public
+ * @deprecated Will be removed soon
```

Comment:
```suggestion
 * @deprecated Will be removed in 5.0.0.
```

Change in all components

Reply:
Done